handlers/add_student.go

```diff
@@ -0,0 +1,34 @@
+package handlers
+
+import (
+	runtime "github.com/ahab94/ApnaSchool"
+	"github.com/ahab94/ApnaSchool/gen/restapi/operations"
+	"github.com/ahab94/ApnaSchool/models"
+	"github.com/go-openapi/runtime/middleware"
+)
+
+// NewAddStudent handles request for saving student
+func NewAddStudent(rt *runtime.Runtime) operations.AddStudentHandler {
+	return &addStudent{rt: rt}
+}
+
+type addStudent struct {
+	rt *runtime.Runtime
+}
+
+// Handle the add student request
+func (d *addStudent) Handle(params operations.AddStudentParams) middleware.Responder {
+	student := models.Student{
+		ID:         params.Student.ID,
```

Comment:
ID should not be in params for post call.. we should restrict ID in post payload.